src/Auth/AuthUtils.php

```diff
@@ -47,6 +48,7 @@ public static function parseJwtToken(string $authToken): object
         try {
             $payload = $exploded[1];
             $payload = JWT::jsonDecode(JWT::urlsafeB64Decode($payload));
+            $payload->storage = null;
```

Comment:
Explicitly not supporting storage client for legacy tokens unless I hear otherwise. I could regex one of the other endpoints to calculate this, but I'm not sure we want to keep supporting the old tokens?